docker/README.md

```diff
@@ -37,18 +48,27 @@ containers to communicate with each other - it prevents worker to connect to Web
 
 on host machine.
 
-## Create directory structure
+**Create directory structure**
+
+In case you want to have the big files (isos and disk images, test and needles) outside of the docker volume container,
+you should create this file structrure from within the directory you are going to execute the container.
 
     mkdir -p data/factory/{iso,hdd} data/tests
 
-It is also necessary to either run all containers in privileged mode, or set selinux properly:
+It could be necessary to either run all containers in privileged mode, or set selinux properly. If you are having problems with it, run this command:
 
     chcon -Rt svirt_sandbox_file_t data
 
 ## Run the Data & WebUI containers
 
+**All data within the volume data container**
+
+    # openSUSE
+    docker run -d -h openqa_data --name openqa_data -v `pwd`/data/factory:/data/factory -v `pwd`/data/tests:/data/tests slindomansilla/openqa_data
+    docker run -d -h openqa_webui --name openqa_webui --volumes-from openqa_data -p 80:80 -p 443:443 slindomansilla/openqa_webui
```

Comment:
now you are replicating the block just because of the name "fedoraqa"? there should be less duplication. There might be the possibility to use the same docker hub project/user